necronomicon.lua

```diff
@@ -79,20 +79,50 @@ function necronomicon(include_slabs)
     end
 end
 
+function necronomicon_world(include_slabs)
```

Comment:
could you fix the indentation of this function? it should be four spaces per indent, no tabs